markdown_stoplight/scopes.md

```diff
@@ -9,80 +9,70 @@ Here's the list of the scopes we provide and what they're used for.
+| `ordr:u`                          | [Update order note](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/orders/order-note-update), [Update billing address](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/orders/order-update-billing-address), [Update shipping address](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/orders/order-update-shipping-address), [Create invoice](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/orders/order-update-billing-address), [Send invoice](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/orders/order-send-invoice) |
+| `ordp:r`                          | [List order processes](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/orders/get-order-processes)              |
+| `paym:m`                          | Required for the use case ["Integrate a payment solution"](https://beyond.docs.stoplight.io/use-cases-543263/#integrate-a-payment-solution)     |
+| `prad:c`                          | [Create product attribute definition](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/product-attribute-definitions/product-attribute-definitions-create-string)         |
+| `prad:r`                          | [List product attribute definitions](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/product-attribute-definitions/product-attribute-definitions-list), [Show product attribute definition details](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/product-attribute-definitions/product-attribute-definition-get) |
+| `prad:d`                          | [Delete product attribute definition](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/product-attribute-definitions/product-attribute-definitions-delete)  |
+| `prat:c`                          | Create product attribute, [Create product attribute of type NUMBER](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/products/product-attribute-create-number-with-name)             |
+| `prat:r`                          | [List product attributes for a product](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/products/product-attributes-list), [Show product attribute details of type NUMBER](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/products/product-attribute-get-number)   |
+| `prat:u`                          | [Update product attribute](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/products/product-attribute-update)             |
+| `prat:d`                          | [Delete product attribute](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/products/product-attribute-delete)               |
+| `prda:r`                          | [Show variation availability details](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/products/variation-availability-get) |
```

Comment:
Why VARIATION and not the normal product one? https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/products/product-availability-get